src/test/kotlin/com/github/imflog/schema/registry/tasks/register/RegisterTaskActionTest.kt

```diff
@@ -337,6 +337,7 @@ class RegisterTaskActionTest {
         val subjects = listOf(
             Subject("test", "${schemaPath}testSubject.avsc", "AVRO")
                 .addLocalReference("B", "${schemaPath}testType.avsc")
+                .setRuleSet("${schemaPath}testRuleSet.json")
```

Comment:
I think we should add a dedicated test that includes testing on the ruleSet, metadata and normalization.